ui/ember-cli-build.js

```diff
@@ -33,9 +18,20 @@ module.exports = function(defaults) {
       plugins: ['transform-object-rest-spread'],
     },
     'ember-cli-babel': {
-      includePolyfill: true,
+      includePolyfill: isProd,
+    },
+    hinting: isTest,
```

Comment:
The build server won't show eslint errors anymore - but with the husky PR we'll get them in the pre-commit hook.

Reply:
that seems fine, since we should also still get eslint warnings in our editors, correct?

Reply:
Yep! You'd only see them before if you were looking at the app build output (which can scroll by pretty fast depending on what you're doing). Editor integration + husky hooks + tests failing lint errors will still catch things.